conbench/app/batches.py

```diff
@@ -43,6 +43,9 @@ def get(self, batch_id):
             self.flash("Error getting benchmarks.")
             return self.redirect("app.index")
 
+        # TODO: paginate
+        benchmarks = benchmarks["data"]
```

Comment:
For now I made this quick fix (and the corresponding changes in `app/results.py` and `app/runs.py`) to get the test suite green. However, I think this will limit these webapp pages to only showing the last 100 matching results. That's not super desirable so I want to look into how we can take better advantage of pagination on these webpages.

Reply:
1. Should we just be querying the db directly instead of hitting the API here?
2. We could at least set the `page_size` to 1000 for the moment to probably nobody is affected

Reply:
Yeah we should just hit the DB directly. I think this is the spirit of https://github.com/conbench/conbench/issues/968.

In an effort to keep PRs reviewable, I'll probably do this in a followup PR. It's okay if staging is "broken" in the meantime. I'll bump page size to 1000 like you suggested.

>we could link the datatables pagination to API pagination, but that would be a pain and mess up searching.

This was my original plan but you're right, we'd really have to rethink how searching works. It would probably be annoying to break search. Especially for those Arrow runs with 3300 results.